cmd/share.go

```diff
@@ -268,9 +269,13 @@ func buildShareOptions(cmd *cobra.Command) options.Options {
 	}
 
 	debug := viper.GetBool("debug")
+	disableinfologs := viper.GetBool("disableinfologs")
 	if debug != true {
 		log.SetLevel(log.InfoLevel)
 	}
+	if disableinfologs == true {
+        log.SetOutput(ioutil.Discard)
```

Comment:
is this formatted correctly?

Reply:
not a good idea to do this. Use log levels to control what is printed and how it is printed. What log is not required as info change it to debug level.